<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="32887" started-at="2019-04-01T20:09:39Z" finished-at="2019-04-01T20:10:12Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="32887" started-at="2019-04-01T20:09:39Z" finished-at="2019-04-01T20:10:12Z">
      <class name="test.LoginOrangeHRMTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.LoginOrangeHRMTest@18be83e4]" name="setUp" is-config="true" duration-ms="4588" started-at="2019-04-01T20:09:39Z" finished-at="2019-04-01T20:09:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="login()[pri:0, instance:test.LoginOrangeHRMTest@18be83e4]" name="login" duration-ms="27513" started-at="2019-04-01T20:09:44Z" finished-at="2019-04-01T20:10:11Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[LOGIN SUCCESSFUL expected:<OrangeHRM tool> but was:<OrangeHRM>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: LOGIN SUCCESSFUL expected:<OrangeHRM tool> but was:<OrangeHRM>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
at test.LoginOrangeHRMTest.login(LoginOrangeHRMTest.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.LoginOrangeHRMTest@18be83e4]" name="tearDown" is-config="true" duration-ms="752" started-at="2019-04-01T20:10:11Z" finished-at="2019-04-01T20:10:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.LoginOrangeHRMTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
